dw-dialog.js

```diff
@@ -20,7 +20,8 @@ export class DwDialog extends LitElement {
         :host {
           display: block;
           outline:none;
-          color: var(--mdc-theme-text-primary); 
+          color: var(--mdc-theme-text-primary);
+          z-index: 999;
```

Comment:
Why do we need to set `z-index` to such high value? Probably, I heard from Ruchita that `mdc-dialog` manages proper z-index in case of even multiple (nested) dialogs.

And what's point of setting `z-index` for `host` element? Actual elements of `dialog` are local doms of this slement